resources/profiles/Creality/machine/Creality K1 Max (0.4 nozzle).json

```diff
@@ -13,7 +13,7 @@
 	"printable_area": ["0x0", "300x0", "300x300", "0x300"],
 	"printable_height": "300",
 	"nozzle_type": "hardened_steel",
-	"auxiliary_fan": "0",
+	"auxiliary_fan": "1",
```

Comment:
Good catch.  I must've just checked the box in the UI and didn't check for the K1 and Max to be consistent.